source/installation/install-on-an-unattended-server/debian-server-installation/install-debian-server.txt

```diff
@@ -1,3 +1,116 @@
+and is exposes it as a web application.
+
+Steps
+-----
+
+1. Download the latest ``.deb`` binary from the `release page <https://migrator-installer-repository.s3.ap-southeast-2.amazonaws.com/index.html>`__.
+
+#. Install Relational Migrator.
+
+   .. code-block:: bash
+
+      sudo apt install ./mongodb-relational-migrator_X.X.X-1_amd64.deb
+
+#. Change the directory to the Relational Migrator ``app/lib`` location.
+
+   .. code-block:: bash
+
+      cd /opt/mongodb-relational-migrator/lib/app/lib
+
+#. (Optional) To connect to Oracle or MySQL drivers, download and place the drivers 
+   into your Relational Migrator directory.
+
+   1. To download and install MySQL or Oracle drivers, see the following:
+
+      - `MySQL: latest 8.0.x version, Platform Independent Version <https://dev.mysql.com/downloads/connector/j/>`_
+
+      - `Oracle: 21.6.0.0 of odjbc11.jar from the Oracle 21c <https://www.oracle.com/database/technologies/appdev/jdbc-downloads.html>`_
+
+   2. Copy the driver files to ``/opt/mongodb-relational-migrator/lib/app/lib``.
+
+      To copy the driver, use the following example: 
+
+      .. code-block:: bash
+      
+         curl https://download.oracle.com/otn-pub/otn_software/jdbc/216/ojdbc11.jar -O -L -b -o
+
+   For more information, see :ref:`Install on a Single Machine <single-machine-install>`.
+
+#. Change the directory to the Relational Migrator ``/bin`` location and start the application.
+
+   .. code-block:: bash
+
+      cd /opt/mongodb-relational-migrator/bin
+
+      ./mongodb-relational-migrator
+
+#. Enable Relational Migrator to run as an unattended standalone server.
+
+   In your ``user.properties`` file, uncomment ``spring.profiles.active: unattended``. 
+
+#. (Optional) Update TLS/SSL configuration properties. 
+
+   In your ``user.properties`` file, uncomment and update TLS/SSL configuration 
+   properties.
+
+   It is recommended that the unattended server installation is supplemented with 
+   TLS/SSL configuration because Relational Migrator can be bound to any IP and port on 
+   the server. For additional information, see `TLS/SSL Configuration Properties <https://docs.spring.io/spring-cloud-skipper/docs/2.8.4/reference/htmlsingle/#configuration-security-enabling-https>`__.
+
+#. (Optional) Update the port that Relational Migrator runs on.
+
+   By default, Relational Migrator is configured to run on port 8278. To change 
+   the port Relational Migrator runs on (for example to port 80 for HTTP or port 443 for 
+   HTTPS), update the ``server.port`` line in your ``user.properties`` file to specify 
+   your preferred port.
+
+#. Run Relational Migrator using the binary.
+
+   The ``bin`` directory can typically be found at ``/opt/mongodb-relational-migrator/bin``.
+   To run Relational Migrator using the binary, run the following:
+
+   .. code-block:: bash
+
+      cd /opt/mongodb-relational-migrator/bin
+
+      ./mongodb-relational-migrator
+
+#. Check logs for errors or other information. 
```

Comment:
(nit) 
This is usually marked (Optional).

Reply:
Done.